CHANGELOG.md

```diff
@@ -2,13 +2,16 @@
 
 ## unreleased
 
+* [FEATURE] Remote-Write: Add sender and receiver support for [Remote Write 2.0-rc.2](https://prometheus.io/docs/specs/remote_write_spec_2_0/) specification #14395 #14427 #14444
+* [ENHANCEMENT] Remote-Write: 1.x messages against Remote Write 2.x Receivers will have now correct values for `prometheus_storage_<samples|histograms|exemplar>_failed_total` in case of partial errors #14444
+
 ## 2.53.1 / 2024-07-10
 
 Fix a bug which would drop samples in remote-write if the sending flow stalled
 for longer than it takes to write one "WAL segment". How long this takes depends on the size
 of your Prometheus; as a rough guide with 10 million series it is about 2-3 minutes.
 
-* [BUGFIX] Remote-write: stop dropping samples in catch-up #14446
+* [BUGFIX] Remote-Write: stop dropping samples in catch-up #14446
```

Comment:
Unrelated change.

Reply:
Related, it was spelled wrongly?

Reply:
happy to kill this, you are right, it's not our item (: